dex/pair/src/locking_wrapper.rs

```diff
@@ -2,17 +2,14 @@ elrond_wasm::imports!();
 
 #[elrond_wasm::module]
 pub trait LockingWrapperModule:
-    crate::config::ConfigModule + token_send::TokenSendModule + pausable::PausableModule
+    crate::config::ConfigModule
+    + token_send::TokenSendModule
+    + permissions_module::PermissionsModule
+    + pausable::PausableModule
 {
-    #[endpoint(setLockingDeadlineEpoch)]
```

Comment:
Why was this and `set_unlock_epoch` removed?

Reply:
According to [the spreadsheet specs](https://docs.google.com/spreadsheets/d/19LhEizrR9nlyPv9yndR83QQZX6FT_03-HhNP1FeLUGg/edit) - both endpoints are not used and should be removed.